challenge_gpsr/README.md

```diff
@@ -22,4 +20,19 @@ or to get multiple commands at once:
 
 ## Test recognition
 
-    rosrun challenge_gpsr test_recognition.py <robot-name>
\ No newline at end of file
+    rosrun challenge_gpsr test_recognition.py <robot-name>
```

Comment:
This is just a test script, so I guess it doesn't have to comply to the launch structure